engine/src/main/java/com/datatorrent/stram/util/SecurityUtils.java

```diff
@@ -87,9 +105,59 @@ public static boolean isHadoopWebSecurityEnabled()
     return hadoopWebSecurityEnabled;
   }
 
-  public static boolean isStramWebSecurityEnabled()
+  @VisibleForTesting
+  protected static boolean isStramWebSecurityEnabled()
   {
     return stramWebSecurityEnabled;
   }
 
+  /**
+   * Setup security related configuration for {@link org.apache.hadoop.yarn.webapp.WebApp}.
+   * @param config
+   * @param sslConfig
+   * @return
+   */
+  public static Configuration configureWebAppSecurity(Configuration config, SSLConfig sslConfig)
```

Comment:
I did not want to use get because the purpose of the method is to "build" the web app security related configuration, which I think "configureXXX" also conveys. To me either is fine, I have higher priority things to work on..